ops/jenkins/bfd-build-eft-sftp-outbound-transfer-lambda/build-push.sh

```diff
@@ -24,18 +24,15 @@ DOCKER_TAG="${DOCKER_TAG_OVERRIDE:-"$BFD_PARENT_VERSION"}"
 DOCKER_TAG_LATEST="${DOCKER_TAG_LATEST_OVERRIDE:-"latest"}"
 
 aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```

Comment:
curious why us-east-1 hard=coded, when next line uses "$AWS_REGION"

Reply:
Good catch, I didn't even notice that. I've replaced it with `$AWS_REGION`.